### ISSUE TITLE
Seed Phrase instead of .bak

### ISSUE THREAD

**Issue by thbestforyourbizdeployment** [DeFiCh/app#1162]

**Comment by prasannavl** [comment]
Not sure why the seed phrase shouldn't be working as expected on ones created with it. But moving to the appropriate repo.

**Comment by izzycsy** [comment]
Hi @thbestforyourbizdeployment , 24 words has been on halt for desktop wallet since v2.3.0, if you're using DeFiChain Desktop app, _wallet passphrase_ and `wallet.dat` are the keys to your wallet — please backup and keep them safe. This Github does not respond to issues regarding saiive wallet, you'll have to go onto their GitHub instead if anything.

Back to your enquiry, the DeFiChain [Light Wallet app](https://github.com/DeFiCh/wallet)'s & [Desktop app](https://github.com/DeFiCh/app)'s restoration methods are irrelevant; meaning to say you can't load the 24 words from mobile app on to the desktop app and neither can you load the `wallet.dat` from the desktop to mobile. If you would like to rely on seed phrase, I'd recommend installing Light Wallet (saving the hassle of running a full node wallet), you'll have to create a new wallet and transfer your funds to the mobile app and continue using from there. 

The team is currently working on integrating the mobile & desktop app — so you can use the same 24 words account concurrently. As for now, these are 2 independent applications. Hope this resolves your issue.


Latest desktop version: [v2.9.3](https://github.com/DeFiCh/app/releases/tag/v2.9.3)

**Comment by thbestforyourbizdeployment** [comment]
Hello @izzycsy thanks for the update. Can I downgrade to a specific version that I can then use my seed phrase?  v2.2.9? 
I need to fix this then move on. i don't have any other way to recover my funds.

**Comment by izzycsy** [comment]
Hi @thbestforyourbizdeployment , you can try but it comes a long with previous bugs & you'll be on another chain, the version before v2.3.0 is [v2.1.17](https://github.com/DeFiCh/app/releases/tag/v2.1.17). If successful, you'll need to update to v2.9.3 and reindex to be on the right chain. 

Alternatively, if you've your `wallet.dat`, I highly suggest that you use it to restore your wallet instead. If you **did not** change Windows PC and did not do anything with DeFi App _except updating_, you'll find the `wallet.dat` here: 

Windows — C: 
` \Users\<username>\AppData\Roaming\DeFi Blockchain\wallets`
 
<img width="823" alt="wallet dat location" src="https://user-images.githubusercontent.com/65014479/160975726-6b7ee9dc-fbad-435f-931a-cf0c3f67cb7b.png">

• In the wallets folder you'll likely see _wallet-bak-123456789_ as well, these are additional backups.
***Note: the wallets folder is a default location and should not be moved**

**Comment by thbestforyourbizdeployment** [comment]
yes, thats the problem, Restored both Linux and Windows machines and didn't save the wallet.dat.

**Comment by thbestforyourbizdeployment** [comment]
Is there a URL to download the node/chain image?

**Comment by thbestforyourbizdeployment** [comment]
Hello @izzycsy 
Is there a URL to download the node/chain image??
 its stuck at 99.34% , 1st time it reached 99.97% then it went down to 99.34%

**Comment by izzycsy** [comment]
> yes, thats the problem, Restored both Linux and Windows machines and didn't save the wallet.dat.

Do you mean you didn't save your `wallet.dat`?

> Is there a URL to download the node/chain image??
> its stuck at 99.34% , 1st time it reached 99.97% then it went down to 99.34%

You can access snapshot via desktop app; #899 . May I know what version are you currently on?

**Comment by izzycsy** [comment]
Hi @thbestforyourbizdeployment , may I know where did you first got your seed phrase generated? On desktop app / Saiive.live?

**Comment by thbestforyourbizdeployment** [comment]
@izzycsy  defichain Native wallet, itv'e been with defichain since 2020

**Comment by izzycsy** [comment]
Hi @thbestforyourbizdeployment , thanks for the reply. Try synchronizing to 100% on [v2.11.0](https://github.com/DeFiCh/app/releases/tag/v2.11.0). When app is synchronized, select `Restore wallet from Recovery Seed`. Beyond that, there's no other way to Restore wallet, it's either via recovery seed or [wallet.dat](https://github.com/DeFiCh/app/discussions/988).

**Comment by thbestforyourbizdeployment** [comment]
@izzycsy @thedoublejay  thanks for the update. The app sync's 100%, using 2.11.0,  but no balance is displayed. The last transaction show is December, however I did use it long after December.  LP is zero balance too.
![balance](https://user-images.githubusercontent.com/73033172/164055790-5470846a-932d-48e3-aecc-98a9a8a3f896.PNG)

**Comment by nico89-exp** [comment]
Have you set a password in the native wallet app? If that your seed phrase cannot recover funds from addresses after setting the password. With setting the password only the wallet.dat can be used to fully recover the funds.

**Comment by thbestforyourbizdeployment** [comment]
@izzycsy @thedoublejay So basically, I am  **screwed** becase I don't have the .dat file? but I still have my seed phrase.
What kind of recovery tech is this? If there is no other way, I will have to stop using this ecosystem.

**Comment by izzycsy** [comment]
Hi @thbestforyourbizdeployment sorry to say but @nico89-exp is right, your seed phrase cannot recover funds from addresses after setting the password. Only with the `wallet.dat` you can.

**Comment by thedoublejay** [comment]
@thbestforyourbizdeployment Try to check the folder of defi/wallets folder. See if there are automatic .dat generated that you can use

**Comment by thbestforyourbizdeployment** [comment]
is there a cmdlet to see the LP address or history so I can add?

**Comment by thbestforyourbizdeployment** [comment]
What does account to account mean? 2 address on the same wallet?
My problem now is that I am not able to see the LP balance. Is there a way to see all my address used before? and see if they have balance?
![image](https://user-images.githubusercontent.com/73033172/167407561-05fa3b3f-07f4-41bd-b3ac-9e5704c3ccf1.png)

**Comment by izzycsy** [comment]
@thbestforyourbizdeployment [accounttoaccount](https://defichain-wiki.com/wiki/Cli_commands#Accounts) refers to a transaction sent from one to another. 

You can add your address to your app if the wallet.dat is yours, toggle off `Automatically generate a new address`. 
<img width="775" alt="Receive   Manually add address" src="https://user-images.githubusercontent.com/65014479/169450206-14007c3d-6b5b-4d0a-9df4-bb83b47245c9.png">
